src/GravityFormsExtensions.php

```diff
@@ -113,11 +113,47 @@ private function hooks(): void
         add_filter('gform_pre_render', [ $this, 'p4_client_side_gravityforms_prefill' ], 10, 1);
         add_filter('gform_form_post_get_meta', [$this, 'p4_gf_enable_default_meta_settings'], 10, 1);
         add_filter('gform_hubspot_form_object_pre_save_feed', [$this, 'p4_gf_hb_form_object_pre_save_feed'], 10, 1);
+        add_action('gform_after_submission', [$this, 'p4_send_gp_pixel_counter'], 10, 2);
 
         add_action('gform_stripe_fulfillment', [ $this, 'record_fulfillment_entry' ], 10, 2);
         add_action('gform_post_payment_action', [ $this, 'check_stripe_payment_status' ], 10, 2);
     }
 
+    /**
+     * Increase in one unit the total number of submissions for a counter by
+     * making an API call to the Greenpeace Pixel Counter, or
+     * adding an iframe.
+     *
+     * @link https://counter.greenpeace.org/documentation
+     * @param array $form The form setting.
+     * @param array $entry A form entry.
+     *
+     */
+    public function p4_send_gp_pixel_counter(array $entry, array $form): void
+    {
+        if (!$form['p4_gf_counter']) {
+            return;
+        }
+
+        $endpoint_url = 'https://counter.greenpeace.org/count';
+        $endpoint_url .= '?id=' . $form['p4_gf_counter'];
+        $endpoint_url .= '&email_hash=' . $this->p4_gf_get_email_hash($form, $entry);
+
+        // Pass the referer explicitily.
+        $wp_remote_get_args = array(
+            'headers' => array(
+                'Referer' => home_url(),
+            ),
+        );
+
+        $response = wp_remote_get($endpoint_url, $wp_remote_get_args);
+
+        GFCommon::log_debug('Pixel Counter Response Message: ' . $response['body']);
```

Comment:
Do we need to keep these `log_debug` calls?

Reply:
They're useful IMO.

Reply:
I see this on the test instance, should we be worried about it?

<img width="1145" alt="Screenshot 2024-10-02 at 10 41 04" src="https://github.com/user-attachments/assets/ddc7e143-3ef4-4c05-8cc7-d037fcdc9fb6">

Reply:
But then will these `log_debug` statements still be useful if we disable logging anyway? 🤔

Reply:
Nevermind, I see now that we have them in other functions too, so it's probably fine.